### PR TITLE
Issue 377

### PR DESCRIPTION
**Pull request by j4sm1ne96**
Add systemProperties.adoc

### CODE REVIEW

File: systemProperties.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+NOTE:
+With boolean properties, see javadoc:
+`net.openhft.chronicle.core.Jvm#getBoolean(java.lang.String)`
+
+When boolean system properties are set to "", "yes" or "true", the property is enabled.
```
Comment (rogersimmons) [inline systemProperties.adoc]: "" - really?
An empty value for a boolean should be an error
@JerryShea - is this expected (or required) behaviour?

Comment (JerryShea) [reply systemProperties.adoc]: I agree that we should not be talking about "" - please see javadoc for net.openhft.chronicle.core.Jvm#getBoolean(java.lang.String, boolean)

We should link to this, and maybe consider pasting some of it in here

**Review by rogersimmons — APPROVED** [review]

File: systemProperties.adoc (hunk 1 above)
Comment (peter-lawrey) [reply systemProperties.adoc]: Currently `-DskipTests` in maven is the same as `-DskipTests=` and `-DskipTests=true`
Are you suggesting the program should fail in the case of `-DskipTest=` ?

Comment (peter-lawrey) [reply systemProperties.adoc]: The code paths for `-DskipTest` and `-DskipTest=` are the same

**Pull request merged**